main.go

```diff
@@ -162,19 +168,6 @@ func main() {
 	log.SetOutput(io.MultiWriter(os.Stderr, logfile))
 	fileOnlyLogger := log.New(logfile, "", log.LstdFlags)
 
-	log.Print("Starting RHMAP System Dump Tool...")
-	log.Print("Preparing tasks...")
```

Comment:
Now that tasks are "prepared" and run simultaneously, we don't need the "Preparing tasks" anymore, we go straight to "Running tasks".
We've introduced the "Preparing tasks" message when the time taken in that step started to be significant. And now it is gone :)